docs/lessons/direction.rst

```diff
@@ -0,0 +1,27 @@
+Direction
+---------
+
+There is a compass on the BBC micro:bit. If you ever make a weather station
+use the device to work out the wind direction.
+
+Compass
++++++++
+
+It can also tell you the direction of North like this::
+
+    from microbit import *
+
+    compass.calibrate()
+
+    while True:
+        needle = ((15 - compass.heading()) // 30) % 12
```

Comment:
I'll add a clarification so those that are interested will be able to work out what's happening on this line, but I think it outside the scope of the documentation to explain it in detail. In fact, working out how it works *IS* quite good fun. ;-)